src/config.jl

```diff
@@ -0,0 +1,76 @@
+export cclibtypes
+
+cclibtypes = Dict(
+    "aonames" => Array{String},
+    "aooverlaps" => Array{Float64, 2},
+    "atombasis" => Array{Array{Int64}},
+    "atomcharges" => Dict{String, Array{Float64}},
+    "atomcoords" => Array{Float64, 3},
+    "atommasses" => Array{Float64},
+    "atomnos" => Array{Int64},
+    "atomspins" => Dict{Array},
+    "ccenergies" => Array{Float64, 2},
+    "charge" => Int64,
+    "coreelectrons" => Array{}, # specify subytpe?
```

Comment:
These types are fine for now but we will definitely want to refine them as we go.